docs/config.rst

```diff
@@ -51,6 +51,13 @@ Configuration Options
   * Default: "localhost"
   * Importance: high
 
+``kafkastore.security.protocol``
```

Comment:
Here's a pull request for this change: https://github.com/confluentinc/docs/pull/103